appveyor.ps1

```diff
@@ -0,0 +1,35 @@
+# the version under development, update after a release
+$version = '3.0.0'
+$modifier = '-beta-6'
+
+function isVersion($s){
+    $v = New-Object Version
+    [Version]::TryParse($s, [ref]$v)
+}
+
+# append the AppVeyor build number as the pre-release version
+if ($env:appveyor){
+    $modifier = $modifier + '-' + [int]::Parse($env:appveyor_build_number).ToString('000')
+    if ($env:appveyor_repo_tag -eq 'true'){
+        $tag = $env:appveyor_repo_tag_name
+        $i = $tag.IndexOf('-')
+        if($i -gt 0)
```

Comment:
Correct. It is currently set up to make builds based on tags. It supports both `3.0.0-beta-5` format and `3.0.0` tags. AppVeyor NuGet packages can be overwritten in case you screw up and want to do it again, just recreate the tag.